Cargo.toml

```diff
@@ -10,8 +10,6 @@ documentation = "http://ironframework.io/doc/router/index.html"
 keywords = ["iron", "web", "http", "routing", "router"]
 
 [dependencies]
+iron = "~0"
```

Comment:
This matches against any 0.x version of iron, which is undesirable as 0.x releases can contain compat-breaking changes according to semver.